crawl-ref/source/initfile.cc

```diff
@@ -319,6 +319,8 @@ const vector<GameOption*> game_options::build_options_list()
         new BoolGameOption(SIMPLE_NAME(tile_menu_icons), true),
         new BoolGameOption(SIMPLE_NAME(tile_filter_scaling), false),
         new BoolGameOption(SIMPLE_NAME(tile_force_overlay), false),
+        new TileColGameOption(SIMPLE_NAME(tile_overlay_col), "#646464"),
```

Comment:
```suggestion
        new TileColGameOption(SIMPLE_NAME(tile_overlay_colour), "#646464"),
```

Reply:
Hi, I don't have problem with the change. But most of the other color options seems to use `_col` instead, that's why I ended using it. Thanks!